bitcoin-magazine/bitcoin-explained/accounts-for-bitcoin-easypaysy!.md

```diff
@@ -0,0 +1,1038 @@
+So that way even if your computer is compromised and only your communication is compromised you still keep your funds.
+So that's why there's two basically.
+
+Sjors: 00:06:35
+
+Yeah and it might be worth reminding the listener that so what you're doing here is you're funding your own or like putting a trivial amount in your own multi-sig address and then immediately spending from that multi-sig address and the reason you have to do the latter is because you reveal the keys when you're spending not when you're receiving.
+So in order for the rest of the world to see what your keys are you have to spend from it.
+
+Aaron: 00:06:58
+
+Right, yeah that's a good addition indeed when you're spending to...
+Well anyway no need to repeat you're right.
+Okay so
+
+Sjors: 00:07:05
+
+Then the OP_RETURN message that's in the outputs of this transaction can communicate any other piece of information to the world.
+
+Aaron: 00:07:12
+
+Yeah, you can basically communicate text in this output.
+So the text we're communicating here is a JSON message, and it includes the instructions for how to pay to this account.
+That's the main thing it includes.
+
+Sjors: 00:07:26
+
+Yeah, I doubt it's actually JSON.
+
+Aaron: 00:07:29
+
+I thought it was JSON.
+
+Sjors: 00:07:30
+
+It's some sort of serialized format, probably, that you can convert to JSON.
+Because otherwise it would be very inefficient.
+
+Aaron: 00:07:37
+
+What Jose told me is it's a small JSON document.
+You think that's probably a simplification?
+
+Sjors: 00:07:44
+
+Well I don't know I didn't read it in that level of detail.
+So there's only 80 bytes available in OP_RETURN.
+So you could use those 80 bytes to write as key characters, you know, with an opening bracket and colons and all this stuff.
+But you can also agree on a serialization format that could put way more information in a smaller space.
+
+Aaron: 00:08:03
+
+But you don't need to communicate a lot of data, you just need to tell people which kinds of transactions you're accepting on this account.
+
+Sjors: 00:08:11
+
+Sure, you can do it inefficient if there's not much data, but if you want more data you can do it more efficiently.
+
+Aaron: 00:08:16
+
+Okay, fair enough.
+Well, anyway, so there's three ways of receiving funds on these accounts.
+The first way is not recommended.
+It's to receive payments on your value publicly.
+So now we're back to reusing addresses.
+We don't actually want that, but it's an option.
+
+Sjors: 00:08:35
+
+It's even worse than that, I guess, because you're announcing, hey, this is where the money is going to the whole world.
+And then it's reusing addresses with everyone that you receive money from, plus everyone you're not even receiving money from.
+Right.
+
+Aaron: 00:08:49
+
+Yeah.
+It's bad.
+It's in there because it's just an option for people to use.
+You can choose not to use it, but you know if we're making this protocol, that was Jose's reasoning.
```

Comment:
I understand that here you added the "you know" as a way to represent the original source as best as possible. Generally, we aim to eliminate stammering, filler words ('like,' 'you know',',right?'), and unnecessary non-verbal communication for a more concise and focused transcript